src/main/java/com/google/devtools/build/skydoc/SkydocMain.java

```diff
@@ -289,7 +282,8 @@ public Module eval(
 
     List<AspectInfoWrapper> aspectInfoList = new ArrayList<>();
 
-    Module module = recursiveEval(semantics, label, ruleInfoList, providerInfoList, aspectInfoList);
+    Module module = recursiveEval(semantics, label, label.getRepository().getName(), ruleInfoList,
```

Comment:
It doesn't really matter in this line since the top-level label passed in by the stardoc rule is always a canonical label. I changed this to the main repository name to stay in line with the behavior of other "tools" such as query.